app/controllers/articles_controller.rb

```diff
@@ -1,4 +1,5 @@
 class ArticlesController < ApplicationController
```

Comment:
This seems to have already existed. Same with the `Article` model. It should have been added here, together with the rest of the functionality.